docs/source/workflows/a2a/a2a-cli.md

```
@@ -0,0 +1,161 @@
+
+### Call Agent
+
+Call an A2A agent with a message and get a response. This is useful for quick testing and one-off queries.
+
+**Usage:**
+```bash
+nat a2a client call --url $A2A_SERVER_URL --message "What is 2 + 2?"
+```
+
+**Output example:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)